src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -358,9 +358,9 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
         for (int i = 0; i < numOfRubricChoices; i++) {
             String tableHeaderCell =
                     FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
-                            "${qnIndex}", questionNumberString,
-                            "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", Sanitizer.sanitizeForHtml(rubricChoices.get(i)),
+                            Const.FeedbackQuestion.RUBRIC_QUESTION_INDEX, questionNumberString,
```

Comment:
I don't think these should go into `Const`. If you really want to extract them into constants, put them in the respective `*QuestionDetails`